docs/tutorials/quickstart.md

```diff
@@ -0,0 +1,302 @@
+# Get started with Coder
+
+<div style="padding: 0px; margin: 0px;">
+  <span style="vertical-align:middle;">Author: </span>
+  <a href="https://github.com/EdwardAngert" style="text-decoration: none; color: inherit; margin-bottom: 0px;">
+    <span style="vertical-align:middle;">Edward Angert</span>
+    <img src="https://avatars.githubusercontent.com/u/17991901?v=4" width="24px" height="24px" style="vertical-align:middle; margin: 0px;"/>
+  </a>
+</div>
+September 23, 2024
+
+---
+
+Follow the steps in this guide to install Coder locally or on a VPS, set up a
+workspace, and connect to it from VS Code.
+
+By the end of this guide, you'll have a remote development environment that you
+can connect to from any device anywhere, so you can work on the same files in a
+persistent environment from your main device, a tablet, or your phone.
+
+## Install and start Coder
+
+<div class="tabs">
+
+## Linux/macOS
+
+1. Install Docker:
+
+   ```bash
+   curl -sSL https://get.docker.com | sh
+   ```
+
+   For more details, visit:
+
+   - [Linux instructions](https://docs.docker.com/desktop/install/linux-install/)
+   - [Mac instructions](https://docs.docker.com/desktop/install/mac-install/)
+
+1. Set up the Docker daemon in rootless mode for your user to run Docker as a
+   non-privileged user:
+
+   ```shell
+   dockerd-rootless-setuptool.sh install
+   ```
+
+   Depending on your system's dependencies, you might need to run other commands
+   before you retry this step. Read the output of this command for further
+   instructions.
+
+1. Assign your user to the Docker group:
+
+   ```shell
+   sudo usermod -aG docker $USER
+   ```
+
+1. Install Coder:
+
+   ```shell
+   curl -L https://coder.com/install.sh | sh
+   ```
+
+   - For standalone binaries, system packages, or other alternate installation
+     methods, refer to the
+     [latest release on GitHub](https://github.com/coder/coder/releases/latest).
+
+## Windows
+
+> **Important:** If you plan to use the built-in PostgreSQL database, ensure
+> that the
+> [Visual C++ Runtime](https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist#latest-microsoft-visual-c-redistributable-version)
+> is installed.
+
+1. [Install Docker](https://docs.docker.com/desktop/install/windows-install/).
+
+1. Download the Coder Windows installer (`.msi`) or standalone binary (`.exe`)
+   from the
+   [GitHub repository assets](https://github.com/coder/coder/releases/latest).
+
+   ![Windows setup wizard](../images/install/windows-installer.png)_Windows
+   setup wizard_
+
+   - Alternatively, you can use the
+     [`winget`](https://learn.microsoft.com/en-us/windows/package-manager/winget/#use-winget)
+     package manager to install Coder:
+
+     ```powershell
+     winget install Coder.Coder
+     ```
```

Comment:
To keep the guide short we can only mention winget. What do you think?

Reply:
I like that. + updated to reflect